app/controllers/apply_jobs_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal:true
+
+class ApplyJobsController < ApplicationController
+ 
+  def create
+    if user_signed_in? && (current_user.user_role.include? 'Job Seeker')
+      @apply_job = ApplyJob.create(user_id: params[:user_id].to_i, job_id: params[:job_id].to_i)
+      if @apply_job.valid?
+        @apply_job.save
+        job=Job.find(params[:job_id].to_i)
```

Comment:
create private method and set before filter for create and destroy method

Reply:
done